models/UserModel.ts

```diff
@@ -103,6 +106,10 @@ export class UserModel extends BaseEntity {
     return this.accessType === UserAccessType.MERCH_STORE_DISTRIBUTOR;
   }
 
+  public isResumeOptedIn(): boolean {
+    return this.resumeOptedIn;
+  }
+
```

Comment:
Is there a reason this function is necessary? Can we just rename the field to be `isResumeOptedIn` or `isResumePublic` so that we can avoid defining this function?

Reply:
This function is not necessary and is removed.